### PR TITLE
Upgrade to support Terragrunt 0.32.2

### PR DESCRIPTION
**Pull request by dmattia**
# Pull Request

## Related Github Issues

- Closes https://github.com/transcend-io/terragrunt-atlantis-config/issues/167
- Closes https://github.com/transcend-io/terragrunt-atlantis-config/issues/166

## Description

Upgrades Terragrunt to 0.32.2, supporting the new Terragrunt features.

Notably, this means that this library now supports having multiple `include` blocks. The Readme contains info on how the merge strategy works, but the TLDR is that the last include block wins most of the time if they specify the same locals.

### CODE REVIEW

**Review by michaelfarrell76 — APPROVED** [review]
epic

File: test_examples/multiple_includes/uses_terraform_12/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  path = find_in_parent_folders("use_terraform_12_parent.hcl")
+}
+
+terraform {
+  source = "git::git@github.com:transcend-io/terraform-aws-fargate-container?ref=v0.0.4"
+}
+
+inputs = {
+  foo = "bar"
+}
```
Comment (anotherminh) [inline test_examples/multiple_includes/uses_terraform_12/terragrunt.hcl:11]: your editor doesn't like newlines at the EOF? 🙃

**Review by anotherminh — APPROVED** [review]
happy open source friday!

File: test_examples/multiple_includes/uses_terraform_12/terragrunt.hcl (hunk 1 above)
Comment (dmattia) [reply test_examples/multiple_includes/uses_terraform_12/terragrunt.hcl:11]: 🇺🇸  Vim gives me the freedom to decide 🇺🇸

**Pull request merged**